float32[4] relativeHum
bool[4] sensor_attached